queue-manager/core/src/manager.ts

```diff
@@ -96,15 +102,16 @@ class Manager {
   private events: Events;
   private persistor: Persistor;
   private context: ManagerContext;
-  private isPaused: boolean = false;
```

Comment:
Why you removed the type? it should be typed like other properties (it only has default value.)

Reply:
ESlint automatically removes this type (when saving the file).